style.css

```diff
@@ -253,3 +253,81 @@ section ul {
 .item12 {
   color: #7f8cff;
 }
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/contact\ form\ shapes-mobile@1x.png);
+  background-size: 50%;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  background-position: top 0 right 0;
+  border-top-left-radius: 60px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  margin: 0 13px;
+}
+
+.me {
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  margin-top: 60px;
+}
+
+.mep {
+  color: #ebebff;
+  display: flex;
+  justify-content: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  margin: 10px;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+input {
+  padding: 8px 5px;
+  color: #cfd8dc;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: 8px 0;
+}
+
+textarea {
+  padding: 8px 5px;
+  color: #cfd8dc;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 100px;
+  margin: 5px 0;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
- [ OPTIONAL ] Why dont you make the text in the textarea easily seen to improve the accessibility of your project :pray: 
  
  ![image](https://user-images.githubusercontent.com/92303891/228665758-143f2193-0b1e-4854-9c6e-55a73163c836.png)